src/clr-angular/forms/datalist/datalist.spec.ts

```diff
@@ -43,4 +53,13 @@ export default function(): void {
       expect(this.clarityElement.id).toEqual('custom-id');
     });
   });
+
+  describe('Native Datalist Element', function() {
+    spec(ClrDatalist, TestNativeDatalist, undefined, { providers: [] });
+
+    it('handles native datalist element without any errors', function() {
+      const nativeDatalist: HTMLDataListElement = <HTMLDataListElement>document.getElementById('native-datalist');
```

Comment:
nit: you shouldn't have to double type the value on right and left of assignment

Reply:
TSLint throws

```
TS2741: Property 'options' is missing in type 'HTMLElement' but required in type 'HTMLDataListElement'.
```

and casting was a simple way to address it. I'll look into it more to see what I can do.

Reply:
I made the type less specific to HTMLElement (which is what type `getElementById` seems to return.